jwst/mrs_imatch/mrs_imatch_step.py

```diff
@@ -328,6 +329,31 @@ def _match_models(models, channel, degree, center=None, center_cs='image'):
     #if np.isnan(sigma_data).any():
     #    print('a nan exists in sigma data')
 
+    # MRS fields of view are small compared to source sizes,
+    # and undersampling produces significant differences
+    # in overlap regions between exposures.
+    # Therefore use sigma-clipping to detect and remove sources
+    # (and unmasked outliers) prior to doing the background matching.
+    # Loop over input exposures
+    for i in range(0,len(models)):
+        image = image_data[i]
+        mask = mask_data[i]
+        weight = sigma_data[i]
```

Comment:
I believe this could be written as:
```suggestion
    for image, mask, weight in zip(image_data, mask_data, sigma_data):
```